src/main/java/com/booksaw/betterTeams/util/TeamUtil.java

```diff
@@ -0,0 +1,43 @@
+package com.booksaw.betterTeams.util;
+
+import com.booksaw.betterTeams.CommandResponse;
+import com.booksaw.betterTeams.Main;
+import com.booksaw.betterTeams.Team;
+import org.jetbrains.annotations.Nullable;
+
+
+public class TeamUtil {
+	private TeamUtil() {
+		
+	}
+	
+	public static @Nullable CommandResponse verifyTeamName(@Nullable String teamName) {
```

Comment:
util methods should probably have javadocs :)

Reply:
probably, but I'm just lazy :p

Reply:
Added